core/src/main/java/com/scalar/db/api/AuthAdmin.java

```diff
@@ -56,7 +61,8 @@ default void dropUser(String username) throws ExecutionException {
    * @param namespaceName the namespace name of the table
    * @param tableName the table name
    * @param privileges the privileges
-   * @throws ExecutionException if the user does not exist or the operation fails
+   * @throws IllegalArgumentException if the user does not exist or the table does not exist
```

Comment:
> if the user does not exist or the table does not exist

What about namespace?

Reply:
For this method, we specify the table along with its namespace. If the namespace does not exist, it means the table doesn't exist either. Thanks.